telos-frontend/src/views/journal/Page.jsx

```diff
@@ -63,6 +63,7 @@ const Page = ({ date, widgets, setWidgets }) => {
         WidgetTypes.CLOCK,
         WidgetTypes.YOUTUBE_PLAYER,
         WidgetTypes.WEATHER,
+        WidgetTypes.Steam,
```

Comment:
In addition to the places Jemimah has mentioned, this could also be changed to STEAM for consistency's sake. Otherwise, it looks good to me!

Reply:
Nice spotting! I have resolved the issue.